src/test/java/edu/usf/cutr/grha/test/ChicagoTncDataTest.kt

```diff
@@ -0,0 +1,62 @@
+package edu.usf.cutr.grha.test
+
+import edu.usf.cutr.grha.io.ChicagoTncParser
+import org.junit.Test
+import java.io.File
+import java.io.FileInputStream
+import kotlin.test.assertEquals
+import kotlin.test.assertNull
+
+class ChicagoTncDataTest {
+
+    private val chicagoDataFile = "src/test/resources/Transportation_Network_Providers_reduced_records.csv"
```

Comment:
If these files are on in the resources forlder, they should really be accessed with `classLoader.getResourceAsStream(fileName);`. You could open an input stream this way and then pass that to the parsers.